src/app/views/provisioning/order.controller.coffee

```diff
@@ -18,10 +21,17 @@
         (response) ->
           vm.subscription.product = response
 
-          # Filters the pricing plans not containing current currency
-          vm.subscription.product.product_pricings = _.filter(vm.subscription.product.product_pricings,
-            (pp) -> (pp.pricing_type in PRICING_TYPES['unpriced']) || _.some(pp.prices, (p) -> p.currency == vm.orgCurrency)
-          )
+          # Get all the possible currencies
+          currenciesArray = []
+          _.forEach(vm.subscription.product.product_pricings,
+            (pp) -> _.forEach(pp.prices, (p) -> currenciesArray.push(p.currency)))
+          vm.currencies = _.uniq(currenciesArray)
+          # Set a default currency
+          if vm.currencies.includes(vm.orgCurrency)
+            vm.selectedCurrency = vm.orgCurrency
+          else
+            vm.selectedCurrency = vm.currencies[0]
```

Comment:
This should try the `MnoeConfig.marketplaceCurrency()` before defaulting to the first one.